backend/app/services/execution/request_builder.py

```
@@ -2249,8 +2257,84 @@ def _build_workspace(self, task: TaskResource) -> dict:
+            label_project_id = labels.get("projectId")
+            project_id = int(label_project_id) if str(label_project_id).isdigit() else 0
+
+        if not project_id:
+            return
+
+        project = (
+            self.db.query(Project)
+            .filter(
+                Project.id == project_id,
+                Project.user_id == task.user_id,
+                Project.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Project.is_active:` for truth checks

Replace with `Project.is_active`

(E712)